### PR TITLE
JdkHelper - Added a path validation function

### PR DESCRIPTION
**Pull request by chenlevy24**
Added a path validaton to jdkHelper and changed getJavaHome function to be static

### CODE REVIEW

**Review by andreban — CHANGES_REQUESTED** [review]
This is generally good. I'd rather if we keep the API stable and avoid breaking changes.

**Review by PEConn — COMMENTED** [review]
LGTM % Andre's comments

File: packages/core/src/lib/jdk/JdkHelper.ts [hunk 1]
```diff
@@ -56,29 +58,58 @@ export class JdkHelper {
...
-   * @returns {string} the value for the JAVA_HOME
+   * @param {Config} config The bubblewrap general configuration
+   * @param {NodeJS.Process} process Information from the OS process
    */
-  getJavaHome(): string {
-    if (this.process.platform === 'darwin') {
-      return this.joinPath(this.config.jdkPath, '/Contents/Home/');
-    } else if (this.process.platform === 'linux' || this.process.platform === 'win32') {
-      return this.joinPath(this.config.jdkPath, '/');
+  getJavaHome(jdkPath: string, process: NodeJS.Process): string {
```
Comment (andreban) [inline packages/core/src/lib/jdk/JdkHelper.ts:71]: The old version of the method doesn't have any parameters and uses the jdkPath / process passed as parameter to the config. We want to keep the old signature.

File: packages/core/src/lib/jdk/JdkHelper.ts [hunk 2]
```diff
@@ -56,21 +58,50 @@ export class JdkHelper {
...
-    const runJavaCmd = this.joinPath(this.getJavaHome(), java);
+    const runJavaCmd = this.joinPath(this.getJavaHome(),
+        java);
     return await executeFile(runJavaCmd, args, this.getEnv());
   }
 
   /**
    * Returns information from the JAVA_HOME, based on the config and platform.
-   * @returns {string} the value for the JAVA_HOME
+   * @param {Config} config The bubblewrap general configuration
```
Comment (andreban) [inline packages/core/src/lib/jdk/JdkHelper.ts:68]: Can we change back the comments here?

Comment (chenlevy24) [reply packages/core/src/lib/jdk/JdkHelper.ts:68]: Ah, I missed it

**Review by andreban — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**